rules/linux/credential_access_proc_credential_dumping.toml

```
@@ -23,10 +23,42 @@ references = [
     "https://github.com/huntergregal/mimipenguin",
     "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20781"
 ]
+note = """## Triage and analysis
+
+### Investigating Credential Dumping via Proc Filesystem
+
+The rule identifies execution of the mimipenguin exploit script password-cracking tool which is linux adaptation of
+Windows tool mimikatz.Mimipenguin exploit script is used to dump clear text passwords from a currently logged-in user.
+The tool exploits a known vulnerability CVE-2018-20781. Malicious actors can exploit the cleartext credentials in memory
+by dumping the process and extracting lines that have a high probability of containing cleartext passwords.
+
+#### Possible investigation steps
+
+- Identify the session entry leader and session user.
+- Examine the contents of session leading to the mimipenguin exploit script execution via the Session View.
+- Examine the command execution pattern in the session, which may lead to suspicious activities.
+- Examine the command execution post mimipenguin exploit to indicate any exfiltration of clear text passwords.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+
+### False positive analysis
+
+- The exploit tool execution can only be a check to assert or test security measures by authorised system administrators.
```

```suggestion
- The exploit tool can be used by authorized system administrators to assert or test security measures.
```
Feel free to ignore if I misunderstood the intended meaning.